errors/css-global.md

```diff
@@ -16,8 +16,7 @@ Or, [update your component to use local CSS (Component-Level CSS) via CSS Module
 
 Consider the stylesheet named [`styles.css`](https://nextjs.org/docs/basic-features/built-in-css-support#adding-a-global-stylesheet)
 
-```jsx
-//styles.css
+```css filename="styles.css"
```

Comment:
Why is this one updating the file name but the others under `errors` are still using a comment?

Reply:
I started converting them before I realized these are just `.md` files instead of `.mdx` files like the rest of the docs. I thought I backed them all out, so good catch. I'll revert it. @delbaoliveira and I just discussed these this morning and will put together a separate effort to get errors / messages converted.